packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -63,6 +66,17 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             bytes memory
         )
     {
+        // Decode the L2 gas limit. It is scaled and serialized in the lower
+        // order bits of transaction.gasLimit. See:
+        // https://github.com/ethereum-optimism/optimism/blob/0c18e1903f8a33a607f782c0081a8fb5071b71ef/l2geth/rollup/fees/rollup_fee.go#L56
+        uint256 gasLimit = SafeMath.mul((_transaction.gasLimit % 10000), 10000);
```

Comment:
should this be called `subCallGasLimit` or something differentiating from `_transaction.gasLimit`?